docs/materials/day4/part1-ex3-blast-split.md

```diff
@@ -12,7 +12,7 @@ Setup
 -----
 
 -   Make sure you are still logged into `training.osgconnect.net`
--   Make sure you are in the directory named `thur-blast-data` under the `stash` filesystem.
+-   Make sure you are in the directory named `thur-blast-data` under your local scratch directory
```

Comment:
how big is the data? if the individual files are big, it should go into stash. if they're small, local scratch is the right place...

Reply:
The original test data is about 100MB but in this exercise we're splitting it into 100 1MB files to act as the inputs for the jobs.